### ISSUE TITLE
Doorbell support?

### ISSUE THREAD

**Issue by jahnkenneth** [swttt/com.swttt.homekit#154]
Would it be possible to make the app compatible with doorbells?
I have a Ring Pro connected to Homey, and would love to have in the Home app. 
I understand that the video feed is not possible, but maybe the Doorbell type is?

**Comment by robertklep** [comment]
Can you post the info for such a device from [this page](https://developer.athom.com/tools/devices) here? I'd like to know which capabilities it supports.

**Comment by jahnkenneth** [comment]
Sorry for the formatting...

Property	Value
ID	fe506456-dab1-4917-9799-e5e938cb31cc
Class	sensor
Driver	homey:app:com.ring — doorbell
Ready	Yes
Available	Yes
Warning	No
Custom icon	No

Capabilities
Capability	Type	Value	Edit value	Last Updated
alarm_generic	boolean	false		3 days ago
alarm_motion	boolean	false		38 minutes ago
measure_battery	number	100		6 minutes ago

**Comment by robertklep** [comment]
Thanks for that!

Can you find out which of the capabilities is used when the door bell button is pressed? I assume `alarm_generic` but want to make sure :)

**Comment by robertklep** [comment]
Hmm, @nhassink just reminded me that device class `doorbell` with capability `alarm_generic` is already supported by Homeykit, so are you sure that your device isn't working?

In Homekit they should appear as Motion Sensor.

**Comment by jahnkenneth** [comment]
Yes, it’s alarm_generic. :-)
Yes, the motion sensor is visible in HomeKit, but I want the alarm_generic button to be represented by a doorbell accessory in Homekit. :-)

**Comment by robertklep** [comment]
Yeah I think it makes sense to present it as a doorbell. Not sure what would happen if I change the Homekit type to something else (if Homekit will see it as a new device, in which case other users may run into problems if they have Homekit automations depending on it).

**Comment by jahnkenneth** [comment]
So it’s a no go..?

**Comment by robertklep** [comment]
Not yet, but I have to do some testing first.

**Comment by jahnkenneth** [comment]
Sign me up for the testing 👍🏻
